pkg/state/state.go

```diff
@@ -1527,6 +1581,10 @@ func (s *stateManager) addBlocks() (*proto.Block, error) {
 	if err := s.stor.handleStateHashes(height, ids); err != nil {
 		return nil, wrapErr(ModificationError, err)
 	}
+	// Store total amount and rewards for each of new blocks.
```

Comment:
Initially I thought that we won't save reward and totalWavesAmount for every block. It's a huge waste of space. For the first 1_740_000 blocks on MainNet it's only zeroes. And after that we don't have to store it for every block We have to store new value of reward only if it's changed. From 0 to 6 on activation of Feature 14 and so on. For now it will be only one value. 
The total Waves amount can be calculated.